lesson7/MyDevice.java

```diff
@@ -0,0 +1,17 @@
+package lesson7;
+
+public class MyDevice implements Volume, State {
```

Comment:
Hm, where do you invoke this class? Please, create Task3 class and demonstrate it there.